src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/DbCellValue.cs

```diff
@@ -22,6 +22,11 @@ public class DbCellValue
         /// </summary>
         public bool IsNull { get; set; }
 
+        /// <summary>
+        /// The file extension to be used for any XML cell values representing an execution plan.
+        /// </summary>
+        public string ExecutionPlanFileExtension { get; set; }
```

Comment:
I think this what we're trying to avoid with some of the comments on the ADS PR, of special casing execution plan too much into the contract.  Having a execution plan file extension attached to every cell of data in a resultset doesn't seem like a flexible, generic enough design.  I wonder if there is a way to have something like "result set auxiliary data" that says something like "this column contains a value that should be displayed with this render" and the auxiliary data is only returned in the case some subset of the data needs special handling in the UI.